packages/cln_plugin/test/cln_plugin_method_test.dart

```diff
@@ -95,6 +113,52 @@ void main() {
       expect(getManifest.containsKey('dynamic'), true);
       expect(getManifest.containsKey('hooks'), true);
       expect(getManifest.containsKey('subscriptions'), true);
+
+      expect(getManifest["rpcmethods"], isNotNull);
+      expect(getManifest["options"], isNotNull);
+      expect(getManifest["notifications"], isNotNull);
+      expect(getManifest["dynamic"], isNotNull);
+      expect(getManifest["hooks"], isNotNull);
+      expect(getManifest["subscriptions"], isNotNull);
+    });
+  });
+  group('Init test: ', () {
+    test('Property test - option registered.', () {
+      var plugin = Plugin(
+          dynamic: true,
+          onInit: (plugin) {
+            return Future.value({});
+          });
+      var request = {
+        "options": {
+          "greeting": "World",
+          "number": [0]
+        },
+        "configuration": {
+          "lightning-dir": "/home/user/.lightning/testnet",
+          "rpc-file": "lightning-rpc",
+          "startup": true,
+          "network": "testnet",
+          "feature_set": {
+            "init": "02aaa2",
+            "node": "8000000002aaa2",
+            "channel": "",
+            "invoice": "028200"
+          },
+          "proxy": {"type": "ipv4", "address": "127.0.0.1", "port": 9050},
+          "torv3-enabled": true,
+          "always_use_proxy": false
+        }
+      };
+      plugin.registerOption(
+          name: 'greeting',
+          type: 'string',
+          def: 'World',
+          description: 'This is the option description.');
+      plugin.init(plugin, request);
+      expect(plugin.options.containsKey('greeting'), true);
+      expect(plugin.options["greeting"]?.name.toString(), "greeting");
+      expect(plugin.options["greeting"]?.value.toString(), "World");
```

Comment:
```suggestion
      expect(plugin.options["greeting"]!.name.toString(), "greeting");
      expect(plugin.options["greeting"]!.value.toString(), "World");
```

you are checking that this is null before so you can add this assertion